app-crypt/adcli/files/adcli-0.9.1-library-include-endian.h-for-le32toh.patch

```diff
@@ -0,0 +1,26 @@
+From b3c559cf7e1d188443eea171ff6935a1e972ad2f Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to the bug at the top of the patch as well as a link to the upstream commit and report if it exists.

Reply:
there was no upstream bug, i did add the commit sha of the upstream commit to the backport

Reply:
Usually I just link to it.